app/models/task/TaskCreationParameters.scala

```diff
@@ -5,7 +5,7 @@ import models.user.Experience
 import play.api.libs.json.{Format, Json}
 
 case class TaskParameters(
-    taskTypeId: String,
+    taskTypeIdOrSummary: String,
```

Comment:
could you add a comment here explaining that after initial processing this will only contain ids?